### ISSUE TITLE
[package] fftw/3.3.9: Missing SIMD options in the recipe

### ISSUE THREAD

**Issue by ruilvo** [conan-io/conan-center-index#4668]
### Package and Environment Details 
  * Package Name/Version: **fftw/3.3.9**
  * Operating System+version: **Windows 10**
  * Compiler+version: **MSVC 2019**

### Steps to reproduce 
So, I'm using these settings in my `conanfile.txt`
```
[requires]
fftw/3.3.9

[options]
fftw:shared=False
fftw:openmp=True
fftw:threads=True
fftw:combinedthreads=True
fftw:precision=single
```
which are not part of any readily available package, so conan goes to compile it from source. The problem is that the compiled output does not make use on any SIMD instructions. 
In the CMakeLists present on `...fftw\3.3.9\_\_\source\source_subfolder` I can see the following:
```
option (ENABLE_SSE "Compile with SSE instruction set support" OFF)
option (ENABLE_SSE2 "Compile with SSE2 instruction set support" OFF)
option (ENABLE_AVX "Compile with AVX instruction set support" OFF)
option (ENABLE_AVX2 "Compile with AVX2 instruction set support" OFF)
```

But these options are not part of the recipe, which only offers:
```
    options = {"shared": [True, False],
               "fPIC": [True, False],
               "precision": ["double", "single", "longdouble"],
               "openmp": [True, False],
               "threads": [True, False],
               "combinedthreads": [True, False]}
```
And for a library like this, this is massively important.

**Comment by uilianries** [comment]
Please, take a look on my answer: https://github.com/conan-io/cmake-conan/issues/311#issuecomment-785016025

It would be great if you could open a PR with that change.

We have other recipes that can be used as example for your case: https://github.com/conan-io/conan-center-index/blob/689e2663fe41689259f61a5dbd3fe245e24c03b9/recipes/gf-complete/all/conanfile.py#L27

**Comment by ruilvo** [comment]
I'll give it a shot. Thanks.

**Comment by SpaceIm** [comment]
Looking at fftw CMakeLists, there should not be auto value, just `[True, False]`.

**Comment by ruilvo** [comment]
I fixed it with PR#4671. It was more work than I expected, but seems to be working.

**Comment by ruilvo** [comment]
Closed by #4671
